cmd/ledgerconfig/deletecmd/deletecmd.go

```diff
@@ -127,7 +127,7 @@ func (c *command) run() error {
 		return err
 	}
 
-	_, err = ch.Execute(req, channel.WithTargetEndpoints(c.Context().Peers...))
+	_, err = ch.Execute(req)
```

Comment:
The option isn't needed anymore?

Reply:
No. The config chaincode is now a user chaincode that specifies a policy. So, the targets should be resolved by the Discovery service according to the policy.